eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/NavigationHeader.js

```diff
@@ -1,18 +1,19 @@
 import React from "react";
 import styled from "styled-components";
 import PropTypes from "prop-types";
+import { withRouter } from "react-router";
 
 import CustomPropTypes from "custom-prop-types";
 import HomeIcon from "./icon-home.svg?inline";
 import MetadataIcon from "./icon-metadata.svg?inline";
 
 import QuestionnaireSettingsModal from "App/QuestionnaireSettingsModal";
-import MetadataModal from "App/MetadataModal";
-import Button from "components/buttons/Button";
+
 import RouteButton from "components/buttons/Button/RouteButton";
 import IconText from "components/IconText";
-import gql from "graphql-tag";
 
+import gql from "graphql-tag";
```

Comment:
Small nit but can we move this to the top import block, just looks a little nicer to sort imports by external dependencies.

Reply:
Done